cmake/external/spdlog.cmake

```diff
@@ -12,11 +12,12 @@ ExternalProject_Add(
 	URL_MD5 a966eea01f81551527853d282896cb4d
 	CMAKE_ARGS
 	-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+	-DCMAKE_POSITION_INDEPENDENT_CODE=ON
```

Comment:
@vigsterkr had to add this to get the fmt static library working

Reply:
yep you are right... sorry that slipped my attention.